installer/release.sh

```diff
@@ -218,6 +221,23 @@ test -z "$GITCONFIG_PATH" || {
 	LIST="$(echo "$LIST" | grep -v "^$GITCONFIG_PATH\$")"
 }
 
+# ARM64 Windows handling
+IS_ARM64=
+if test -n "$arm64_artifacts_directory"
+then
+	echo "Including ARM64 artifacts from $arm64_artifacts_directory";
+	IS_ARM64=1
+	# If directory starts with a slash (i.e. /arm64), strip it, because InnoSetup can't handle it.
+	if [[ $arm64_artifacts_directory == *\/* ]];
+	then
+		ARM64_FILES=$(find ${arm64_artifacts_directory} -type f | cut -c2-)
```

Comment:
You probably have a much cleaner solution, @dscho 😊 it does work with this setup though. The installer is created successfully and includes the ARM64 artifacts 🚀